### DISCUSSION TITLE
thanhle.blog/fix-loi-force-layout-reflow-anh-huong-toi-performance-frontend

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#59] [Ideas]
# thanhle.blog/fix-loi-force-layout-reflow-anh-huong-toi-performance-frontend

https://thanhle.blog/blog/fix-loi-force-layout-reflow-anh-huong-toi-performance-frontend

**Comment by hai123vn** [comment]
chừng nào anh ra bài mới nữa vậy anh ơi, em đang rất hóng

**Comment by trinhvanminh** [comment]
mấy ảnh trên notion mất tiêu r anh ơi
